luxonis_train/core/trainer.py

```diff
@@ -63,6 +63,9 @@ def graceful_exit(signum: int, _):
             self._upload_logs()
 
             if self.cfg.tracker.is_mlflow:
```

Comment:
What if TensorBoard or WandB would be used? Does it handle transition to fail automatically?